components/actionbar/index.css

```diff
@@ -104,24 +64,35 @@
 
 	/* item counter */
 	.spectrum-FieldLabel {
-		--mod-fieldlabel-font-size: var(--spectrum-action-bar-counter-font-size);
-		--mod-fieldlabel-line-height: var(--spectrum-actionbar-item-counter-line-height);
-		color: var(--mod-actionbar-item-counter-color, var(--spectrum-actionbar-item-counter-color));
-		padding-inline-start: var(--mod-actionbar-close-button-to-counter, var(--spectrum-actionbar-close-button-to-counter));
+		font-size: var(--spectrum-action-bar-counter-font-size);
+		line-height: var(--spectrum-line-height-100);
+
+		color: var(--spectrum-neutral-content-color-default);
+		padding-inline-start: var(--spectrum-action-bar-close-button-to-counter);
 
 		/* cjk language support */
 		&:lang(ja),
 		&:lang(zh),
 		&:lang(ko) {
-			line-height: var(--mod-actionbar-item-counter-line-height-cjk, var(--spectrum-actionbar-item-counter-line-height-cjk));
+			line-height: var(--spectrum-cjk-line-height-100);
 		}
 	}
 
 	/* action group */
 	.spectrum-ActionGroup {
 		/* align to end by default */
 		margin-inline-start: auto;
-		padding-inline-start: var(--mod-actionbar-spacing-label-to-action-group, var(--spectrum-actionbar-spacing-label-to-action-group)); /* space between label and action group */
+		padding-inline-start: var(--spectrum-action-bar-label-to-action-group-area);
+	}
+}
+
+.spectrum-ActionBar--emphasized {
```

Comment:
This will be a good place to explore how to set these nested component styles without passthroughs in 2nd gen.